source/new.rst

```diff
@@ -5,78 +5,107 @@ New API Guidelines
-* Useful: APIs that land in Firefox should be useful since all APIs have a maintenance burden.
-* High level: WebExtensions provide a known public API layer on top of Mozilla code, allowing the underlying code to change quicker and easier.
-* Alternatives: Do alternatives exist for example, WebAPIs?
+* **Security**: the API should not expose an unacceptable risk to the end user.
+* **Privacy**: similar to above.
+* **Performance**: APIs should be async by default and focus on not causing jank, hangs or any sort of bad performance in Firefox.
+* **Multi-process**: all APIs need to be multi-process aware.
+* **Useful**: APIs that land in Firefox should be useful since all APIs have a maintenance burden.
+* **High level**: WebExtensions provide a known public API layer on top of Mozilla code, allowing the underlying code to change quicker and easier.
+* **Alternatives**: Do alternatives exist for example, WebAPIs?
 
 Compared to legacy Firefox add-ons there is a significant sandbox around WebExtension APIs that is very restrictive. Access to priviledged APIs within Firefox, preferences, arbitrary File System access, socket access and so on are restricted (at the time of writing this documentation). APIs that break out of this sandbox will need very clear justification for why.
 
-Triaging APIs
--------------
+General principles
+------------------
 
-If you file a bug asking for an API it will go through the bug triage process. Since this is useful to how APIs land, here's what currently happens in bug triages:
+As we've been triaging bugs, we've come up some general principles and guidelines. These are liable to change and adapt as we go along.
 
-Weekly new bug triage
-~~~~~~~~~~~~~~~~~~~~~
+Users over Developers
++++++++++++++++++++++
 
-We look at all new bugs in an attempt to spot serious bugs, regressions or other issues. We try to give each bug a priority and mark as *[triaged]* in the whiteboard. The point here is to do an initial triage and catch critical things. We also label bugs that might be good for contributors or need thinking about. The latter are marked *[design-decision-needed]*, but its important to point out that straightforward change or obvious bugs just go through.
+One example was the request to prevent users from dragging `browserAction buttons into the overflow menu <https://bugzilla.mozilla.org/show_bug.cgi?id=1314070>`_, tab menu or customize menu. That's because its the users choice where they place their buttons, *not* the developers.
 
+Users understanding UI changes
+++++++++++++++++++++++++++++++
 
-Bi-weekly community meeting
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Where possible we should try to make sure that changes to Firefox are due to extensions. If a UI change is going to occur, it should occur on install of the extension or in a user action. Where possible that UI that is added should have some clear trace back to the extension that installed it, maybe through icons or text.
```

Comment:
I don't understand this, did you mean something like "...make sure that changes to Firefox made by extensions are clear to the user"?